src/components/Searchbar/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import './Searchbar.css';
+import { getCity } from '../../redux/searchbar/searchbarSlice';
+
+const Searchbar = () => {
```

Comment:
@franclobo, hey can you please send me a video for that because I tried it again and it's working, I don't understand what it's not working on your side?

Reply:
Hey @wassimchakib could we join a Zoom meeting now?